tests/puppeteer/pages/FO/home.js

```diff
@@ -19,11 +22,39 @@ module.exports = class Home extends CommonPage {
   }
 
   /**
-   * Go the product page
+   * Go to the product page
    * @param id, product id
    */
   async goToProductPage(id) {
     await this.page.waitForSelector(this.logoHomePage, {visible: true});
     await this.waitForSelectorAndClick(this.productImg.replace('%NUMBER', id), 5000);
   }
+
+  /**
+   * Get number of products displayed in all products page
+   * @return integer
+   */
+  async getNumberOfProducts() {
+    const productNumber = await this.getTextContent(this.totalProducts);
+    const numberOfProduct = /\d+/g.exec(productNumber.match(/Il y a ([0-9]+)/)).toString();
```

Comment:
We don't have a selector for this one ? Isn't the FO supposed to be in english ?

Reply:
No the FO is in Frensh because we installed PS with country France.
No there is no selector for the product number :/

Reply:
I don't think we need text for this one 
https://github.com/PrestaShop/PrestaShop/pull/15389#discussion_r320329378

Reply:
Yes I fixed it ;)